test/Test_LabeledTuple.re

```diff
@@ -0,0 +1,98 @@
+open Alcotest;
+open Haz3lcore;
+
+let test_rearrange = (name, analyzed_types, actual_values, expected_values) =>
+  test_case(
+    name,
+    `Quick,
+    () => {
+      let actual =
+        LabeledTuple.rearrange2(
+          ~show_b=[%derive.show: int],
+          analyzed_types,
+          actual_values,
+        );
+      check(
+        list(pair(option(string), int)),
+        name,
+        expected_values,
+        actual,
+      );
+      ();
+    },
+  );
+// Create a property test
+let tests: list(test_case(return)) = [
```

Comment:
I needed these tests to make sure the function is working as expected. We don't need it for this PR but it's worth adding some property tests:

- Unlabeled values don't rearrange
- Fully labeled tuple types arbitrarily rearrange fully labeled values
- There's probably some properties around duplicates, and mismatched lengths but I haven't thought them through yet.